### ISSUE TITLE
Document a lesson to setup linting rules

### ISSUE THREAD

**Issue by willklein** [willklein/testing-dx#11]
Could be to make the buttons accessible

Plugins/rules to setup:
- https://www.npmjs.com/package/eslint-plugin-jsx-a11y

Preparation:
- setup ESLint config in advance without the rules configured
- practice opening dev tools to show the missing accessibility role
- have a bad test that looks up a button by test id

Exposition:
- setup a rule and fix a problem
- have others setup the rule, fix another problem
- change a test to lookup by role
